upstream-sha256: 7ff91f09fc250d661e7d6828692fd94c2d3bc670079b1c477c24a839e2728d89
actual-sha256: c0271ddf54797e087973efd69656d1b68afa751d6df52956a0ca5113ad5fd677
--- upstream.expected
+++ tolk.actual
@@ -15,7 +15,7 @@
       float val4 = (*(data4_16+(Lidx3+4)));
       float val5 = (*(data4_16+(Lidx3+8)));
       float val6 = (*(data4_16+Lidx3));
-      *(data0_16+(alu0+Lidx3)) = ((*(buf0+0))+(val0[0]*val6)+(val0[1]*val4)+(val0[2]*val5)+(val0[3]*val3));
+      *(data0_16+(alu0+Lidx3)) = ((*(buf0+0))+(0.0f+(val0[0]*val6)+(val0[1]*val4)+(val0[2]*val5)+(val0[3]*val3)));
     }
   }
 }
@@ -62,7 +62,7 @@
     float alu24 = ((*(buf2+1))+(*(buf1+1)));
     float alu25 = ((*(buf2+2))+(*(buf1+2)));
     float alu26 = ((*(buf2+3))+(*(buf1+3)));
-    *(buf3+0) = ((*(buf3+0))+(alu23*alu23)+(alu24*alu24)+(alu25*alu25)+(alu26*alu26));
+    *(buf3+0) = ((*(buf3+0))+((alu23*alu23)+(alu24*alu24)+(alu25*alu25)+(alu26*alu26)));
   }
   *(data0_1+0) = ((*(buf0+0))+(*(buf3+0)));
 }
